src/components/header-footer.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom'
+
+import { StyledBtns, StyledNav } from '../Styles';
+
+function Header() {
+   return (
+      <>
+         <header className="header">
+            <h1>Randomized Character Creator</h1>
+
+            <nav>
```

Comment:
Since this nav is being used in both the header and footer it can be it's own component as well